network/epcommon/endpoint_common.go

```diff
@@ -28,8 +28,9 @@ RFC for Link Local Addresses: https://tools.ietf.org/html/rfc3927
 */
 
 const (
-	enableIPForwardCmd = "sysctl -w net.ipv4.ip_forward=1"
-	enableIPV6Cmd      = "sysctl -w net.ipv6.conf.all.disable_ipv6=%d"
+	enableIPForwardCmd   = "sysctl -w net.ipv4.ip_forward=1"
+	enableIPV6Cmd        = "sysctl -w net.ipv6.conf.all.disable_ipv6=%d"
```

Comment:
Potentially rename to toggleIPV6, given that cmd usage could be to enable or disable